src/arch/linux/Makefile.main

```diff
@@ -76,10 +76,16 @@ $(top_builddir)/etc/Xfonts/%.pcf.gz: $(REALTOPDIR)/etc/%.bdf | $(top_builddir)/e
 
 $(LIBS_): | $(top_builddir)/lib
 
+ifeq ($(OS),Darwin)
 $(BINPATH)/bin/$(DOSBIN): $(LIBS_)
+	$(LD) $(ALL_LDFLAGS) $(DOSBIN_LDFLAGS) -o $@ \
+	   -Wl,-all_load $(LIBS_) $(LIBS) \
+	   -Wl,-map,$(BINPATH)/bin/dosemu.map -Wl,-pagezero_size,0x10000
```

Comment:
Oh, how about finishing the jit
patch to avoid this MAP_32BIT... :(

Reply:
I had that finished :) But never sure why it wasn't accepted, there was a place that added and subtracted mem_base (for rep string instructions) but I didn't understand why that was not good?

Reply:
I don't remember.
Please make a comment to the patch,
and I'll re-hash the discussion.

Reply:
IIRC the patch was removing mem_base,
but in some places it was left.

Reply:
so I left the `-Wl,-pagezero_size,0x10000` for now, not sure what's still blocking #1844

Reply:
> so I left the -Wl,-pagezero_size,0x10000 for now, not sure what's still blocking

The lack of response to simple
questions, and instead silent
marking of them as resolved?

Reply:
I answered all the questions, no silent resolves, but if my answer is not complete feel free to ask for clarification.

Reply:
Maybe you have problems with
github then. Aren't your answers
marked as "Pending" or alike?

Reply:
Since this is not for the first time,
but only in #1844 , github problems
look much more plausible that anything
else. :)
Make sure you are not in a review
mode in that ticket.

Reply:
yes.. ah, I need to make them visible to you! This Pending confused me a little.

Reply:
Yes, and all the prev comments
(a few months ago) also didn't arrive.
This "Pending" is hell confusing.
I wonder why github still doesn't
do anything about this.